src/lib/create-instance.js

```diff
@@ -13,14 +13,23 @@ import extractOptions from '../options/extract-options'
 import deleteMountingOptions from '../options/delete-mounting-options'
 import createFunctionalComponent from './create-functional-component'
 
+import { logEvents } from '../lib/log-events'
+
 export default function createConstructor (
   component: Component,
-  options: Options
+  options: Options,
+  eventCaptor: EventCaptor
 ): Component {
   const mountingOptions = extractOptions(options)
 
   const vue = mountingOptions.localVue || createLocalVue()
 
+  vue.mixin({
+    beforeCreate: function () {
+      logEvents(this, eventCaptor.emitted, eventCaptor.emittedByOrder)
```

Comment:
I actually think we should use `this.__emitted` and `this.__emittedBy`, because otherwise (I think) all events logged by any of the child components will be added to the eventCaptor. Using `this.__emitted` means we can support child component emitted, which won't work at the moment.

Reply:
Yeah, I think you are right, if a `localVue` instance will be reused, the captor might be added several times. I thought it might be better to keep it non-intrusive, but I will do it using `this.__emitted` then.

Reply:
> Using this.__emitted means we can support child component emitted, which won't work at the moment.

How would that work? Is there anything required to implement this as well? Right now this does not get captured by `wrapper.emitted()`:
```js
    const wrapper = mount({
      template: '<child-component></child-component>',
      components: {
        'child-component': {
          beforeCreate: function () {
            this.$emit('childEvent')
          }
        }
      }
    })
```

Reply:
Sorry, that wasn't very clear. I meant that at the moment (in this PR) only root components will support emitted, child components won't, because when the VueWrapper is created with them, they won't have an eventCaptor to pass in. 

This should work:
```js
    const wrapper = mount({
      template: '<child-component></child-component>',
      components: {
        'child-component': {
          beforeCreate: function () {
            this.$emit('childEvent')
          }
        }
      }
    })
wrapper.find(childComponent).vm.emitted('childEvent') // returns array

Thinking about the underscore, I think it's fine to have a single underscore.